05/app.js

```diff
@@ -1,3 +1,32 @@
 console.log('DOM');
 
 const curr = document.querySelector('.js-curr');
+
+if(curr) {
+    const articleElement = curr.parentElement;
+    const buttonElement = document.createElement("button");
+    buttonElement.innerText = "Usuń z koszyka";
+    articleElement.appendChild(buttonElement);
+
+    if(articleElement.hasChildNodes()) {
+        const child = articleElement.children;
```

Comment:
Uwaga na nazwę zmiennej `child` to liczna pojedyncza - a jest to tablica. Będzie to mylące, bo tam mamy wiele elementó więc powinna być liczna mnoga.